src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +98,77 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     * @param string|null $locale
+     * @param int|null    $maxRows
+     * @param int|null    $startRow
+     * @return Collection
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = null, int $startRow = null): Collection
+    {
+        $url = sprintf(self::BASE_ENDPOINT_URL, 'countryInfoJSON', $this->username);
+
+        if (isset($country)) {
+            $url = sprintf('%s&country=%s', $url, $country);
+        }
+
+        if (isset($maxRows)) {
+            $url = sprintf('%s&maxRows=%d', $url, $maxRows);
+        }
+
+        if (isset($startRow)) {
+            $url = sprintf('%s&startRow=%d', $url, $startRow);
+        }
+
+        $url = sprintf('%s&style=FULL', $url);
+
+        if (null !== $locale) {
+            // Locale code transformation: for example from it_IT to it
+            $url = sprintf('%s&lang=%s', $url, substr($locale, 0, 2));
+        }
+
+        $content = $this->getUrlContents($url);
+        if (null === $json = json_decode($content)) {
+            throw InvalidServerResponse::create($url);
+        }
+
+        $data = $json->geonames;
+
+        if (empty($data)) {
+            return new AddressCollection([]);
+        }
+
+        $results = [];
+        foreach ($data as $item) {
```

Comment:
Yes, if the country codes requested are multiple (like `IN,US` instead of just `IN` or even if the country code is omitted altogether,) the result is multiple!

Sample: http://api.geonames.org/countryInfoJSON?lang=it&country=IN,US&username=demo&style=full
(Use my username if it reaches daily limit.)